eng/build.ps1

```diff
@@ -66,6 +66,17 @@ if ($PSVersionTable.PSVersion.Major -lt "5") {
     exit 1
 }
 
+function Check-MaxPath {
+    $regOutput = reg query HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem /v LongPathsEnabled
```

Comment:
Can this be done using `Get-ItemProperty -Path HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem\LongPathsEnabled`?

Reply:
Yes, yes it can. Good call, I didn't particularly like using string comparison for `reg query` output.